core-server/src/main/java/org/glassfish/jersey/server/ServerRuntime.java

```diff
@@ -669,11 +669,18 @@ public OutputStream getOutputStream(final int contentLength) throws IOException
 
             } catch (final Throwable ex) {
                 if (response.isCommitted()) {
```

Comment:
I can refactor that, but in any case, if it is IOException it will throw up the exception (it does not matter whether the response was committed or not).

Reply:
I think if there is IOException it does not really matter if the response was sent or not, because we cannot respond. Probably we need to skip the finally block in any case for this type of exception.